src/Equinor.ProCoSys.IPO.Command/InvitationCommands/UpdateRfocAcceptedStatus/UpdateRfocAcceptedCommandHandler.cs

```diff
@@ -90,10 +94,39 @@ public async Task<Result<Unit>> Handle(UpdateRfocAcceptedCommand request, Cancel
             var commPkgNos = certificateCommPkgsModel.CommPkgs.Select(c => c.CommPkgNo).ToList();
             var mcPkgs = certificateMcPkgsModel.McPkgs.Select(mc =>new Tuple<string, string>(mc.McPkgNo, mc.CommPkgNo)).ToList();
             _invitationRepository.UpdateRfocStatuses(project.Name, commPkgNos, mcPkgs);
+            var certificate = await GetOrCreateCertificateAsync(request.ProCoSysGuid, project, cancellationToken);
```

Comment:
This new logic should have unit test(s) in src/tests/Equinor.ProCoSys.IPO.Command.Tests/InvitationCommands/UpdateRfocAccepted/UpdateRfocAcceptedCommandHandlerTests.cs